WordPress/openverse
# Implement inverted CC icons for dark mode

## Fixes



Fixes #4909 by @fcoveram

## Description



This PR adds inverted icons for CC licenses. A small cleanup was performed for the added SVGs (such as removing unnecessary attributes and merging paths).

## Testing Instructions


0. Check out the PR and run the dev server.
1. Search for something to go to the search page.
   - Note the icons.
2. Switch to dark mode.
3. Check the following:
   - The license icons in the filters sidebar should be reversed.
   - The license icons in image and audio results should be reversed.
4. Switch back to light mode.
5. Click on any media item to go to the single result page.
   - Note icons.
6. Switch to dark mode.
7. Check the following:
   - The license icons in the license explanation should be reversed.

## Checklist


      PRs) or the media properties generator (`ov just catalog/generate-docs media-props`
      for the catalog or `ov just api/generate-docs` for the API) where applicable.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
